state-backend-template

```diff
@@ -3,6 +3,7 @@ terraform {
   backend "s3" {
     bucket = "terraform-state"
     key    = "simphera.tfstate"
-    region = var.region
+    #The region of the bucket (same region as your deployment, ie. var.region).
+    region = "eu-central-1"

```

Comment:
We should use partial backend configuration maybe: 
```hcl
terraform init -backend-config="region=eu-central-1"
```
```diff
backend "s3" {
-    region = var.region
    #The region of the bucket (same region as your deployment, ie. var.region).
+    region = "" //to set the region, use the following command: terraform init -backend-config="region=eu-central-1"
}
```


more info: https://developer.hashicorp.com/terraform/language/settings/backends/configuration#partial-configuration

Reply:
Yes, this is one of possible options